driver_cpl/shr/seq_infodata_mod.F90

```diff
@@ -60,6 +63,27 @@ MODULE seq_infodata_mod
    character(len=*), public, parameter :: seq_infodata_orb_variable_year    = 'variable_year'
    character(len=*), public, parameter :: seq_infodata_orb_fixed_parameters = 'fixed_parameters'
 
+   ! Type to hold pause/resume signaling information
+   type seq_pause_resume_type
+      private
+      logical                :: atm_pause = .false. ! atm write pause restart file
+      logical                :: lnd_pause = .false. ! lnd write pause restart file
+      logical                :: ice_pause = .false. ! ice write pause restart file
+      logical                :: ocn_pause = .false. ! ocn write pause restart file
+      logical                :: glc_pause = .false. ! glc write pause restart file
+      logical                :: rof_pause = .false. ! rof write pause restart file
+      logical                :: wav_pause = .false. ! wav write pause restart file
+      logical                :: cpl_pause = .false. ! cpl write pause restart file
+      character(SHR_KIND_CL) :: atm_resume(num_inst_atm) = ' ' ! atm resume file
+      character(SHR_KIND_CL) :: lnd_resume(num_inst_lnd) = ' ' ! lnd resume file
+      character(SHR_KIND_CL) :: ice_resume(num_inst_ice) = ' ' ! ice resume file
+      character(SHR_KIND_CL) :: ocn_resume(num_inst_ocn) = ' ' ! ocn resume file
+      character(SHR_KIND_CL) :: glc_resume(num_inst_glc) = ' ' ! glc resume file
+      character(SHR_KIND_CL) :: rof_resume(num_inst_rof) = ' ' ! rof resume file
+      character(SHR_KIND_CL) :: wav_resume(num_inst_wav) = ' ' ! wav resume file
+      character(SHR_KIND_CL) :: cpl_resume = ' '               ! cpl resume file
+   end type seq_pause_resume_type
+
```

Comment:
Creating a sub-type for pause/resume means that all that data does not have to be allocated for other runs.